salt/orch/update.sls

```diff
@@ -57,9 +57,10 @@ update_modules:
   salt.function:
     - tgt: {{ master_id }}
     - name: cmd.run
-    - bg: True
     - arg:
-      - systemctl reboot
+      - sleep 15; systemctl reboot
```

Comment:
@inercia on a 50 node cluster, 1 minute will be nearly an hour delay in the upgrade - hence trying to get a smaller number than 1 minute.